extensions/schema-compare/src/dialogs/schemaCompareDialog.ts

```diff
@@ -182,7 +183,7 @@ export class SchemaCompareDialog {
 
 			this.sourceTextBox = this.view.modelBuilder.inputBox().withProperties({
 				value: this.schemaCompareMainWindow.sourceEndpointInfo ? this.schemaCompareMainWindow.sourceEndpointInfo.packageFilePath : '',
-				width: 275,
+				width: 280,
```

Comment:
make this a const so it can be reused and it'll be easier to update in just one place in the future.